index.html

```diff
@@ -477,20 +511,3 @@
   </section>
 </body>
 </html>
```

Comment:
Now that we split `PerformanceObserverEntryList` into a separate section, there is a bunch of empty definitions and we're missing the previous text (not sure where or how we lost it). 

> The  interface provides the same getEntries, getEntriesByType, getEntriesByName methods as the Performance interface, except that PerformanceObserverEntryList operates on the observed emitted list of events instead of the global timeline.